src/ImageSharp/ColorSpaces/Companding/SRgbCompanding.cs

```diff
@@ -18,49 +22,115 @@ namespace SixLabors.ImageSharp.ColorSpaces.Companding
     /// </remarks>
     public static class SRgbCompanding
     {
+        private const int Length = Scale + 2; // 256kb @ 16bit precision.
+        private const int Scale = (1 << 16) - 1;
+
+        private static readonly Lazy<float[]> LazyCompressTable = new Lazy<float[]>(() =>
+        {
+            var result = new float[Length];
+
+            for (int i = 0; i < result.Length; i++)
+            {
+                double d = (double)i / Scale;
+                if (d <= (0.04045 / 12.92))
+                {
+                    d *= 12.92;
+                }
+                else
+                {
+                    d = (1.055 * Math.Pow(d, 1.0 / 2.4)) - 0.055;
+                }
+
+                result[i] = (float)d;
+            }
+
+            return result;
+        });
```

Comment:
```suggestion
        }, LazyThreadSafetyMode.PublicationOnly);
```

Reply:
Ok, realized it's unnecessary, since writing `float[]` (`object`) should be atomic.